packages/api/src/client/internal/generated/openapi/request/OpenApiRequest.ts

```diff
@@ -0,0 +1,11 @@
+export type OpenApiRequest<TExpectedResponse, TResponse = TExpectedResponse> = (
+    endpointName: string,
+    method: string,
+    endpointPath: string,
+    data?: any,
+    headers?: { [header: string]: string | number | boolean | undefined | null; },
```

Comment:
I think custom headers are pretty rare right? Certainly more rare than queryArguments. Given arguments can be omitted in generated code, we can slim things down a bit by ordering these by frequency of use.